check/src/typecheck.rs

```diff
@@ -748,7 +748,10 @@ impl<'a> Typecheck<'a> {
 
                 let mut duplicated_fields = FnvSet::default();
                 for field in associated_types.iter().chain(fields.iter()) {
-                    if self.error_on_duplicated_field(&mut duplicated_fields, span, field.0.clone()) {
+                    if
+                        self.error_on_duplicated_field(&mut duplicated_fields,
```

Comment:
Still weird formatting, maybe I accidently added a newline and rustfmt is not removing it now?